lib/country_select/tag_helper.rb

```diff
@@ -13,9 +13,8 @@ def country_option_tags
         option_tags = options_for_select(priority_countries_options, option_tags_options)
         option_tags += html_safe_newline + options_for_select([priority_countries_divider], disabled: priority_countries_divider)
 
-        if priority_countries_options.map(&:second).include?(option_tags_options[:selected])
-          option_tags_options[:selected] = nil
-        end
+        option_tags_options[:selected] = [option_tags_options[:selected]] unless option_tags_options[:selected].kind_of?(Array)
```

Comment:
This seems to coincidentally work... I'm not sure that that's necessarily bad, but it seems like `option_tags_options[:selected]` should never be transformed into an array.

```ruby
if option_tags_options[:selected].kind_of?(Array)
  option_tags_options[:selected].delete_if { |selected| priority_countries_options.map(&:second).include?(selected) }
elsif priority_countries_options.map(&:second).include?(option_tags_options[:selected])
  option_tags_options[:selected] = nil
end
```

Or if we're fine with the coincidence of `selected` being an array for single selects then it becomes much simpler.

```ruby
option_tags_options[:selected] = Array(option_tags_options[:selected]).reject do |selected|
  priority_countries.include?(selected)
end
```

And beyond that then we can just cast `option_tags_options[:selected]` to an array immediately and shorten this operation a bit.

```ruby
    def country_option_tags
      option_tags_options = {
        :selected => Array(@options.fetch(:selected) { value(@object) }),
        :disabled => @options[:disabled]
      }

      if priority_countries.present?
        option_tags_options[:selected].delete_if { |selected| priority_countries.include?(selected) }

        priority_countries_options = country_options_for(priority_countries, false)

        option_tags = options_for_select(priority_countries_options, option_tags_options)
        option_tags += html_safe_newline + options_for_select([priority_countries_divider], disabled: priority_countries_divider)

        option_tags += html_safe_newline + options_for_select(country_options, option_tags_options)
      else
        option_tags = options_for_select(country_options, option_tags_options)
      end
    end
```

I'm just not sure why `selected` being cast into Array still allows single selects to work correctly, but I think I'd be okay with merging it that way if all the tests pass since travis gives us good coverage across rails versions.